facade/interface.go

```diff
@@ -48,6 +49,14 @@ type FacadeInterface interface {
 
 	ScheduleService(ctx datastore.Context, serviceID string, autoLaunch bool, desiredState service.DesiredState) (int, error)
 
+	StartService(ctx datastore.Context, request dao.ScheduleServiceRequest) (int, error)
```

Comment:
Rather than expand the surface area of Facade with functions that aren't used outside of Facade, I would move more of the code in the publicendpointresource into the facade - see note below